libcontainer/rootfs_linux.go

```diff
@@ -957,6 +963,11 @@ func mountPropagate(m *configs.Mount, rootfs string, mountLabel string) error {
 		flags &= ^unix.MS_RDONLY
 	}
 
+	isRo := m.Flags&unix.MS_RDONLY == 1
+	if isRo && m.Device == "tmpfs" {
+		flags &= ^unix.MS_RDONLY
```

Comment:
In fact you don't have to check the flag -- you can just go ahead and clear it.

Makes sense to add a comment explaining why are we doing it here.

Reply:
I removed the flag check and added a comment.